source/ranecu.f90

```diff
@@ -73,6 +73,38 @@ subroutine ranecu(rvec,len,mcut)
 
 end subroutine ranecu
 
+! Init the random generator, that is, set seeds with proper value checks as described in [2]
+subroutine recuinit(is1,is2)
+
+  use crcoall
+
+  integer,           intent(in) :: is1
+  integer, optional, intent(in) :: is2
+
+  if(is1 < 1 .or. is1 > 2147483562) then
```

Comment:
Make these magic numbers into parameters (constants) for the module, then reference them by name?

Reply:
I did it this way for readability and consistency since every other constant in this module is done the same way. The same values are also used in the generator.

This way, at least, you can do a direct comparison to the values described in the paper if you want to check the code.